health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/BreadCrumbNew.js

```
@@ -0,0 +1,140 @@
+const BreadCrumbNew = (props) => {
+  const [expanded, setExpanded] = useState(false);
+  const [crumbsToDisplay, setCrumbsToDisplay] = useState([...props?.crumbs]);
+
+  useEffect(() => {
+    if (props?.maxItems && props?.crumbs.length > props?.maxItems && !expanded) {
+      const startCrumbs = props.crumbs.slice(0, props?.itemsBeforeCollapse || Math.ceil(props.maxItems / 2));
+      const endCrumbs = props.crumbs.slice(
+        -1 * (props?.itemsAfterCollapse || Math.floor(props.maxItems / 2))
+      );
+
+      let updatedCrumbs = startCrumbs.concat(
```

Biome

[error] This let declares a variable that is only assigned once.

'updatedCrumbs' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)